packages/twenty-front/src/modules/settings/data-model/components/SettingsDataModelNewFieldBreadcrumbDropDown.tsx

```diff
@@ -51,15 +71,25 @@ export const SettingsDataModelNewFieldBreadcrumbDropDown = () => {
   const navigate = useNavigate();
   const location = useLocation();
   const { objectSlug = '' } = useParams();
+  const [searchParams] = useSearchParams();
   const theme = useTheme();
 
+  const fieldType = searchParams.get('fieldType') as SettingsFieldType;
   const isConfigureStep = location.pathname.includes('/configure');
 
-  const handleClick = (isConfigureStep: boolean) => {
-    if (isConfigureStep) {
-      navigate(`/settings/objects/${objectSlug}/new-field/configure`);
+  const handleClick = (step: 'select' | 'configure') => {
+    if (step === 'configure') {
+      if (isDefined(fieldType)) {
+        navigate(
+          `/settings/objects/${objectSlug}/new-field/configure?fieldType=${fieldType}`,
+        );
+      } else {
```

Comment:
We usually avoid else and prefer a structure with return, in this particular case it can be made much more readable :)